src/odemis/driver/test/omicronxx_test.py

```diff
@@ -63,7 +63,7 @@ def test_scan_multi(self):
             dev.terminate()
 
 
-class TestGenericxX(object):
+class TestGenericxX(unittest.TestCase):
```

Comment:
I think this might be a base test class. There is no setup class and if you read the test cases it requires a `self.dev` attribute, which is not available, so the test cases will fail. Now that you let it inherit from `unittest.TestCase` the unittest library will see it as a test class, but it will fail.

You could consider renaming this to something like BaseTestGenericxX, and removing your current changes.

Reply:
I thought these changes where still on another branch. But I will try if it fixes the problem from that other branch. Thanks!